services/provider/proto/provider.proto

```diff
@@ -90,6 +93,12 @@ message StorageConfigResponse{
     repeated ExternalResource externalResource = 1;
     // Contains hash of desired config
     string desiredConfigHash = 2;
+
+    message StorageStatus {
```

Comment:
StorageStatus is a bad name as it indicates that the only thing that gets here is a status and only about the storage  parts of the system, we need a message to hold general/global information that we want to pass

Maybe use `SystemAttributes` as the message name? @rewantsoni WDYT?

Reply:
Yes, `SystemAttributes` would a better suited name